vignettes/articles/buffering.Rmd

```diff
@@ -6,10 +6,12 @@ title: "Buffering"
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.width = 7,
-  fig.height = 5,
+  fig.width = 8,
```

Comment:
I _believe_ this will make formatting consistent across pkgdown and CRAN. That's also why I accepted the pkgdown default below, rather than having shorter figures on CRAN